dev-util/watchman/watchman-4.9.0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 )
```

Comment:
There is an ongoing effort to remove python2 stuff, please try a newer version with python3 support.

Reply:
The problem is that the newer version they have is [extremely building-unfriendly](https://github.com/facebook/watchman/issues/851#issuecomment-680272404), at least without downloading build dependencies during the build which seems incredibly awful (and blocked by the portage sandbox anyway).

One (likely) easy fix I see would be simply patching the python scripts to work on Python 3 since it's just the CLI scripts that are broken...

Reply:
I've added patches for python 3 compatibility (based on the changes from upstream)

Reply:
Maybe those deps can be added to SRC_URI.

Reply:
They don't exist as standalone patches, I got them from the individual git commits between the two tags (minus the ones that just run a python code formatter)

Reply:
I meant regarding the build dependencies you mentioned in the first comment (not the patches you added).